<launch>
<arg name="robot" value="hal"/>
<!-- the RGB files to be played - the folder is, by default, ~/.ros/ -->
<arg name="rgbfiles" default="/media/disk/Datasets/rlpd/frames/*rgb*"/>
<!-- true to see all images and rviz -->
<arg name="display" default="false"/>
<!-- spawn all PPLM -->
<include file="$(find people_recognition_vision)/launch/ukf_multimodal_lite.launch">
  <arg name="robot" value="$(arg robot)" />
  <arg name="pplm_use_euclidean" value="true"/>
  <arg name="pplm_use_face_rec"  value="true"/>
  <arg name="pplm_use_height"    value="true"/>
  <arg name="pplm_use_nite"      value="true"/>
  <arg name="pplm_use_phs"       value="true"/>
  <!-- change face rec index file -->
  <arg name="face_recognizer_pplm_index_file"
       value="$(find vision_utils)/data/images/faces/people_lab/index_rlpd.xml"/>
</include>

  <!-- PPLM benchmarker -->
<remap from="match_ppl" to="$(arg robot)/height_pplm/match_ppl"/>
<!--
<remap from="match_ppl" to="$(arg robot)/euclidean_pplm/match_ppl"/>
<remap from="match_ppl" to="$(arg robot)/face_recognizer_pplm/match_ppl"/>
<remap from="match_ppl" to="$(arg robot)/nite_pplm/match_ppl"/>
<remap from="match_ppl" to="$(arg robot)/phs_pplm/match_ppl"/>
-->
<node pkg="people_recognition_vision" type="pplm_benchmarker.exe"
  name="pplm_benchmarker"  output="screen"  required="true"
  args="_display:=$(arg display) _names:='1;2;3' $(arg rgbfiles)" />
</launch>
